lab-brianna/lib/parse-json.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

A top level comment that you need a README for an easy 2 points. Some other easy points:

include an .eslintrc
include a .gitignore
include a readme with project description
include any npm scripts for starting server, linting, testing, etc